data/scripting/win_conditions/territorial_functions.lua

```diff
@@ -14,6 +14,12 @@ local team_str = _"Team %i"
 local wc_has_territory = _"%1$s has %2$3.0f%% of the land (%3$i of %4$i)."
 local wc_had_territory = _"%1$s had %2$3.0f%% of the land (%3$i of %4$i)."
 
+local wc_stats = "Territory percentage"
+-- This needs to be exactly like wc_points, but localized, because wc_points
```

Comment:
I guess 'wc_points' should be 'wc_stats' here?

Reply:
Yes, it should.